Makefile

```diff
@@ -40,3 +42,27 @@ build:
 .PHONY: install
 install:
 	go install ./cmd/fastly
+
+.PHONY:
+changelog:
+	github_changelog_generator -u fastly -p cli \
+		--no-pr-wo-labels \
+		--no-author \
+		--enhancement-label "**Enhancements:**" \
+		--bugs-label "**Bug fixes:**" \
+		--release-url "https://github.com/fastly/cli/releases/tag/%s" \
+		--exclude-labels documentation \
+		--exclude-tags-regex "v.*-.*"
```

Comment:
This is for "pre" releases and the like?

Reply:
Correct, we only want to document proper releases.